src/main/assembly/fakeroot.xml

```diff
@@ -103,6 +103,12 @@ Installed in a fakeroot directory structure.
             <outputDirectory>/etc/init.d</outputDirectory>
             <fileMode>0755</fileMode>
         </fileSet>
+	<!-- .info files -->
+        <fileSet>
+            <directory>src/main/infofiles</directory>
+            <outputDirectory>/etc/grid-security/certificates</outputDirectory>
```

Comment:
First of all, they might be used by other services, in that sense they're not specific to Argus, hence the suggestion to ship them with the EGI policy packages. For most services they are probably not needed, Argus needs them due to the limitation that we can _not_ make logical OR statements in a permit rule, something along the lines of
```
rule permit {
      vo="atlas"
      { ca-policy-names="policy-igtf-classic" OR
        ca-policy-names="policy-igtf-mics" OR
        ca-policy-names="policy-igtf-slcs" OR
        ca-policy-names="policy-igtf-iota"
      }
  }
```
Furthermore, info files have always been in the same directory as the pem files, that includes policy info files such as policy-igtf-classic.info but also the info files for the CAs themselves.